### PR TITLE
Added the CMakeLists.txt to jsonParseV2

### PR DESCRIPTION
**Pull request by kzangeli**
Added a missing file to last PR (#1071)

### CODE REVIEW

File: src/lib/jsonParseV2/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    parseMetadata.h
+    jsonParseTypeNames.h
+)
+
+
+
+# Include directories
+# -----------------------------------------------------------------
+include_directories("${PROJECT_SOURCE_DIR}/src/lib")
+include_directories("/opt/rapidjson/rapidjson-1.0.2/include")
```
Comment (fgalan) [inline src/lib/jsonParseV2/CMakeLists.txt]: This is very environment specific... The usual place of installing libraries is `/usr/lib` and `/usr/local/lib`.

Comment (kzangeli) [inline src/lib/jsonParseV2/CMakeLists.txt]: That could be discussed.
/usr/lib should never be used.
/usr/local/lib was invented many years ago when hard disks were expensive and we had NFS mounted operating system (if the disk server didn't answer (mounted on /usr/lib), then the local HD was used as backup, in /usr/local/lib).

Libraries that you add manually to your system should be put in /opt (that way (and if you have /opt as a partition) you can replace the whole linux distro without having to touch what you installed by hand).

This is my opinion, it is no absolute truth ...

Comment (kzangeli) [inline src/lib/jsonParseV2/CMakeLists.txt]: Fixed in 9dfe6e3

**Comment by fgalan** [comment]
LGTM

**Pull request merged**